api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -532,7 +533,14 @@ public void setVisit(Visit visit) {
 		
 		return encounterProviders.stream()
 				.filter(ep -> includeVoided || !ep.getVoided())
-				.collect(Collectors.groupingBy(EncounterProvider::getEncounterRole, Collectors.mapping(EncounterProvider::getProvider, Collectors.toSet())));
+				.collect(Collectors.toMap((ep) -> (ep).getEncounterRole(),
```

Comment:
Can I modify it to previous implementation and add a commit in this pull request? @teleivo